frontend/src/pages/account/Security/dialogs/TokenDialog.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+    <ff-dialog ref="dialog" data-el="add-token-dialog" header="Add Token" :confirm-label="token ? 'Save' : 'Create'" :disablePrimary="disableConfirm" @confirm="confirm()">
+        <template #default>
+            <form class="space-y-4" @submit.prevent>
+                <FormRow v-model="input.name" data-form="token-name" :disabled="edit">
+                    Name
+                    <template #description>Token name</template>
```

Comment:
```suggestion
                <FormRow v-model="input.name" data-form="token-name" :disabled="edit">
                    Token name
```

Feels redundant to have both the label and description set to much the same - suggest collapsing down to just the label.